modules/core/src/main/scala/qq/droste/basis.scala

```diff
@@ -10,8 +10,10 @@ import data.list._
 import util.newtypes._
 
 import cats.Eval
-import cats.{Foldable, Monad}
-import cats.kernel.{Monoid, Eq}
+import cats.Eq
+import cats.Foldable
+import cats.Monad
+import cats.Monoid
```

Comment:
Why this change?

Reply:
Just removing the `.kernel` from the imports. In the process I expanded the single line imports, simply because I tend to avoid them.

Reply:
I think we could put those rules in a `.scalafmt.conf` file to reach homogeneicity (I think I spelled it wrong :D )?

Reply:
Yeah that'd be nice! Were you volunteering to take on the effort? 😄

Reply:
https://github.com/andyscott/droste/pull/94 there we go!